specs/granite/receipts_accumulator.md

```diff
@@ -0,0 +1,223 @@
+# Granite - Receipts Accumulator
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Timestamp Activation](#timestamp-activation)
+- [Rationale](#rationale)
+- [Constants & Definitions](#constants--definitions)
+- [Accumulator Tree Construction](#accumulator-tree-construction)
+  - [Interior Commitment Truncation Rules](#interior-commitment-truncation-rules)
+- [Block Execution Changes](#block-execution-changes)
+  - [Header Changes](#header-changes)
+    - [`extraData` size enforcement](#extradata-size-enforcement)
+    - [`extraData` format](#extradata-format)
+    - [Header RLP Size Considerations](#header-rlp-size-considerations)
+  - [Accumulator Tree Functions](#accumulator-tree-functions)
+    - [Insertion](#insertion)
+    - [Verification](#verification)
+- [Eventual Migration to EIP-7685](#eventual-migration-to-eip-7685)
+- [Forwards Compatibility Considerations](#forwards-compatibility-considerations)
+- [Client Implementation Considerations](#client-implementation-considerations)
+- [Infrastructure Considerations](#infrastructure-considerations)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+---
+
```

Comment:
Small bit: most specs have a top level overview section, this just doesnt have a header